content/blogs/2024-04-05-getting-started-with-kestra.md

```diff
@@ -0,0 +1,315 @@
+---
+title: "Getting Started with Kestra"
+description: "This blog post lists 10 Kestra features that have changed my work for the better."
+date: 2024-04-05T10:00:00
+category: Solutions
+author:
+  name: Will Russell
+  image: "wrussell"
+image: /blogs/2024-04-05-getting-started-with-kestra.png
+---
+
+Kestra is an event driven data orchestration platform that’s highly flexible and easy to use. This guide is going to go through the basics and get you started building your own pipeline!
+
+Originating as a platform for data orchestration, Kestra finds itself well equipped for managing all types of CI/CD pipelines with it’s highly flexible interface and huge range of plugin support. Through this blog post, I’m going to show you how to get setup with Kestra and set up a simple workflow to run a Python script every hour and send the result as a Discord notification.
```

Comment:
```suggestion
Originating as a platform for data orchestration, Kestra finds itself well-equipped to manage all types of pipelines with its highly flexible interface and a huge range of plugins. Through this blog post, I’m going to show you how to get setup with Kestra and set up a simple workflow to run a Python script every hour and send the result as a Discord notification.
```

CI/CD is one of the least common use cases, so worth cutting or replacing with data pipelines

Reply:
Thanks for clarifying! I will get this updated!